### DISCUSSION TITLE
1.3.1

### DISCUSSION THREAD

**Discussion by vhanla** [vhanla/winxcorners#62] [Announcements]
**Full Changelog**: https://github.com/vhanla/winxcorners/compare/1.3.0...1.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/vhanla/winxcorners/releases/tag/1.3.1'>1.3.1</a>.</em>

**Comment by srvtsk** [comment]
Them advanced options window always shows in dark theme, despite them PC runs exceptionally with light theme. Switching themes in them system prefs doesn't help.
